sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -150,18 +151,15 @@ public async Task DownloadAsync_ReadFromSecondaryStorage()
             var actual = new MemoryStream();
             await response.Value.Content.CopyToAsync(actual);
             TestHelper.AssertSequenceEqual(data, actual.ToArray());
-            Assert.AreEqual(SecondaryStorageTenantPrimaryHost(), testExceptionPolicy.HostsSetInRequests[0]);
-            Assert.AreEqual(SecondaryStorageTenantSecondaryHost(), testExceptionPolicy.HostsSetInRequests[1]);
+            Assert.AreEqual(SecondaryStorageTenantPrimaryHost(), SimulatedFailures.HostsSetInRequests[0]);
+            Assert.AreEqual(SecondaryStorageTenantSecondaryHost(), SimulatedFailures.HostsSetInRequests[1]);
         }
 
         [Test]
+        [SimulateStorageRequestFailure(new[] { "PUT" })]
```

Comment:
So is the new attribute going to cause all PUT requests in the test to fail once?
I guess this test will still pass because we aren't checking the count of the failures.